### PR TITLE
Fixed another query issue

### PR DESCRIPTION
**Pull request by tonyito**
**Fixed** Issue where items list on all pages despite having different params

### CODE REVIEW

File: server/controllers/mainController.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ mainController.getPageUnique = (req, res, next) => {
 //controller to get list of items from a selected page
 mainController.getList = (req, res, next) => {
   const query = `SELECT items.id AS id, items.name AS item_name, user_id FROM items 
-                INNER JOIN pages ON pages.id = ${res.locals.locationID} `;
+                INNER JOIN pages ON items.page_id = ${res.locals.locationID} AND pages.id = ${res.locals.locationID}`;
   db.query(query)
```
Comment (michaelbayday) [inline server/controllers/mainController.js]: const data = [res.locals.locationID];

Comment (michaelbayday) [reply server/controllers/mainController.js]: db.query(query,data)

File: server/controllers/mainController.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ mainController.getPageUnique = (req, res, next) => {
 //controller to get list of items from a selected page
 mainController.getList = (req, res, next) => {
   const query = `SELECT items.id AS id, items.name AS item_name, user_id FROM items 
-                INNER JOIN pages ON pages.id = ${res.locals.locationID} `;
+                INNER JOIN pages ON items.page_id = ${res.locals.locationID} AND pages.id = ${res.locals.locationID}`;
```
Comment (michaelbayday) [inline server/controllers/mainController.js]: INNER JOIN pages ON items.page_id = $1 AND pages.id = $1`;

**Comment by tonyito** [comment]
Fixed

**Pull request merged**